api/src/Validator/Period/AssertGreaterThanOrEqualToLastScheduleEntryEnd.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Validator\Period;
+
+use Attribute;
+use Symfony\Component\Validator\Constraint;
+
+#[Attribute]
+class AssertGreaterThanOrEqualToLastScheduleEntryEnd extends Constraint {
+    public string $message = 'Due to existing activities, end-date can not be earlier then {{ endDate }}';
```

Comment:
```suggestion
    public string $message = 'Due to existing schedule entries, end-date can not be earlier then {{ endDate }}';
```
"Existing activities" ist auf französisch genauso unklar wie "existing schedule entries": Zumindest im Kurs heisst ein Block "point de cours", nicht "activité". Daher würde ich vom Backend lieber die technisch korrekte englische Nachricht herausgeben, und es dem Frontend durch separate Übersetzungen überlassen, das für Normaluser besser verständlich zu machen.